Blazor data binding doc needs a parameter

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/components/data-binding.md?plain=1#L162

The above line needs to be changed to:

```csharp
private void Set(string value) {}
```

The missing parameter is causing the following errors:

```
...\Microsoft.NET.Sdk.Razor.SourceGenerators\Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator\Pages_BindTest_razor.g.cs(306,240,306,243): error CS1503: Argument 1: cannot convert from 'method group' to 'Func<string, Task>'
...\Microsoft.NET.Sdk.Razor.SourceGenerators\Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator\Pages_BindTest_razor.g.cs(350,420,350,423): error CS1503: Argument 2: cannot convert from 'method group' to 'Action<string>'
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b211df59-e1c9-86c0-c904-31fa3c361e6d
* Version Independent ID: 3f912dfc-5ba5-6c6c-ba93-1ae78508a5cc
* Content: [ASP.NET Core Blazor data binding](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/data-binding?view=aspnetcore-7.0)
* Content Source: [aspnetcore/blazor/components/data-binding.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/components/data-binding.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

--- a/aspnetcore/blazor/components/data-binding.md
+++ b/aspnetcore/blazor/components/data-binding.md
@@ -142,25 +142,29 @@ Examples
 
 <h2>Elements</h2>
 
-<input type="text" @bind:get="text" @bind:set="(value) => { }" />
-
+<input type="text" @bind:get="text" @bind:set="(value) => { text = value; }" />
 <input type="text" @bind:get="text" @bind:set="Set" />
-
 <input type="text" @bind:get="text" @bind:set="SetAsync" />
 
 <h2>Components</h2>
 
-<InputText @bind-Value:get="text" @bind-Value:set="(value) => { }" />
-
+<InputText @bind-Value:get="text" @bind-Value:set="(value) => { text = value; }" />
 <InputText @bind-Value:get="text" @bind-Value:set="Set" />
-
 <InputText @bind-Value:get="text" @bind-Value:set="SetAsync" />
 
 @code {
     private string text = "";
 
-    private void Set() {}
-    private Task SetAsync(string value) { return Task.CompletedTask; }
+    private void Set(string value)
+    {
+        text = value;
+    }
+
+    private Task SetAsync(string value)
+    {
+        text = value;
+        return Task.CompletedTask;
+    }
 }
 ```
 

